src/views/DocumentationTopic.vue

```diff
@@ -197,7 +197,7 @@ export default {
     // hierarchy/breadcrumb for a given topic. We choose to render only the
     // first one.
     parentTopicIdentifiers: ({ topicProps: { hierarchy: { paths: [ids = []] = [] } } }) => ids,
-    navigatorParentTopicIdentifiers: ({ topicProps: { hierarchy: { paths = [] } } }) => (
+    navigatorParentTopicIdentifiers: ({ topicProps: { hierarchy: { paths = [[]] } } }) => (
```

Comment:
Part of the reason I'm hesitant to add these kinds of defaults (for data required to be there by the spec) is that it can be easy to get into a rabbit hole of how far down we have to add these defaults.

For example, should we guard against when there is nothing in `hierarchy`? How about when `hierarchy` is not defined or `null`? Ok, but what about when `references` itself is null? We can guard every single layer if we choose to, but it quickly becomes a lot of added code for situations we shouldn't be hitting and don't always have an obvious fallback UI for.

In this case, we will prevent crashing in some very specific scenarios but the navbar itself will be empty even when that happens, which may be even more confusing for the user. Not saying we shouldn't do this, we just need to be careful about being overly defensive about data that is expected to be present.

Reply:
I understand what you mean. Would you like me to revert this path fallback?

Reply:
no